blocks/init/components/lists/docs/readme.md

```diff
@@ -0,0 +1,14 @@
+# Lists Component
+
+Component used in multiple blocks with all or partial features. This component is named lists because JavaScript has list as a reserved word.
```

Comment:
```suggestion
A component used in multiple blocks with all or partial features. This component is named lists because JavaScript has list as a reserved word.
```